components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -1208,14 +1208,15 @@ func (ins *BackupFromBashtionImp) executeOnRemoteAndPoolStatus(commandString str
 	} else {
 		_, err := sshUtil.connectAndExecuteCommandOnRemoteSteamOutput(commandString)
 		if err != nil {
-			writer.Errorf("error in executing backup commands on Automate node %s,  %s \n", automateIps[0], err.Error())
-			return status.Wrapf(err, status.BackupRestoreError, "error in executing backup commands on Automate node %s", automateIps[0])
+			writer.Errorf("error in executing backup %s commands on Automate node %s,  %s \n", subCommand, automateIps[0], err.Error())
+			return status.Wrapf(err, status.BackupRestoreError, "error in executing backup %s commands on Automate node %s", subCommand, automateIps[0])
```

Comment:
see if this can be written as ->
```suggestion
			return status.Wrapf(err, status.BackupRestoreError, "error in executing \"backup %s\" commands on Automate node %s", subCommand, automateIps[0])
```